sources/packages/api/src/services/application/application.service.ts

```diff
@@ -602,6 +602,34 @@ export class ApplicationService extends RecordDataModelService<Application> {
 
     return application?.offering;
   }
+  /**
+   * Get all active applications of a institution location
```

Comment:
Please correct the grammar to be "Get all active applications of an institution location with application_status is completed."

Reply:
Karma :) lol